app/Http/Controllers/TaskController.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Task;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+
+class TaskController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $tasks = Task::orderBy('created_at', 'asc')->paginate(5);
+        return view('tasks', compact('tasks'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        //
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        $validated = Validator::make($request->all(), [
+            'name' => 'required|max:255'
+        ])->validate();;
+        $task = new Task;
+        $task->name = $request->name;
+        $task->save();
+    
+        return redirect('/');
```

Comment:
```suggestion
        return redirect->route('route_name');
```
tránh dùng url nhé :) dùng route name đi